_config.yml

```diff
@@ -277,3 +277,11 @@ fetch-remote:
       - dest: "build/bake/reference.md"
         src:
           - "docs/bake-reference.md"
+
+  - repo: "https://github.com/docker/scout-cli"
+    default_branch: "main"
+    ref: "main"
```

Comment:
This should probably point to a specific release (instead of `main`). OTOH, I guess the `scout-cli` repository is only updated if we do a new release, so perhaps it's ok to just fetch from `main`.

Reply:
I think that will be possible with the next release.